src/core/tsi/ssl_transport_security.cc

```diff
@@ -853,6 +882,52 @@ static int NullVerifyCallback(int /*preverify_ok*/, X509_STORE_CTX* /*ctx*/) {
   return 1;
 }
 
+static int RootCertExtractCallback(int preverify_ok, X509_STORE_CTX* ctx) {
```

Comment:
Please check whether `ctx` is null when we enter the function. If we call `X509_STORE_CTX_get0_chain` and `ctx` is null, then we will segfault.

Reply:
Will do